docs/src/constants/redirect-routes.js

```diff
@@ -20,124 +20,5 @@ import * as ROUTES from './routes';
-  '/components/native/text-view': ROUTES.TEXT_INPUT,
-  '/components/native/tappable-link-label': ROUTES.LINK,
-  '/components/native/touchable-native-feedback':
-    ROUTES.TOUCHABLE_NATIVE_FEEDBACK,
-  '/components/native/touchable-overlay': ROUTES.TOUCHABLE_OVERLAY,
-  '/components/star-rating-interactive': ROUTES.STAR_RATING,
-  '/components/tappable-link-label': ROUTES.LINK,
-  '/components/text-span': ROUTES.TEXT,
-  '/components/text-view': ROUTES.TEXT_INPUT,
-  '/components/utilities': ROUTES.COMPONENTS,
-  '/components/utilities/alignment': ROUTES.ALIGNMENT,
-  '/components/utilities/theming': ROUTES.THEMING,
-  '/components/web': ROUTES.COMPONENTS,
-  '/components/web/accordions': ROUTES.ACCORDION,
-  '/components/web/autosuggest': ROUTES.AUTOSUGGEST,
-  '/components/web/badge': ROUTES.BADGE,
-  '/components/web/banner-alerts': ROUTES.BANNER_ALERT,
-  '/components/web/barcharts': ROUTES.BARCHART,
-  '/components/web/blockquotes': ROUTES.BLOCKQUOTE,
-  '/components/web/breakpoints': ROUTES.BREAKPOINT,
-  '/components/web/buttons': ROUTES.BUTTON,
-  '/components/web/calendar': ROUTES.CALENDAR,
-  '/components/web/cards': ROUTES.CARD,
-  '/components/web/chips': ROUTES.CHIP,
-  '/components/web/code': ROUTES.CODE,
-  '/components/web/datepicker': ROUTES.DATEPICKER,
-  '/components/web/description-lists': ROUTES.DESCRIPTION_LIST,
-  '/components/web/dialogs': ROUTES.DIALOG,
-  '/components/web/drawer': ROUTES.DRAWER,
-  '/components/web/fieldsets': ROUTES.FIELDSET,
-  '/components/web/forms': ROUTES.FORM,
-  '/components/web/horizontal-grid': ROUTES.HORIZONTAL_GRID,
-  '/components/web/horizontal-nav': ROUTES.HORIZONTAL_NAV,
-  '/components/web/icons': ROUTES.ICON,
-  '/components/web/images': ROUTES.IMAGE,
-  '/components/web/links': ROUTES.LINK,
-  '/components/web/lists': ROUTES.LIST,
-  '/components/web/mobile-scroll-container': ROUTES.MOBILE_SCROLL_CONTAINER,
-  '/components/web/modals': ROUTES.MODAL,
-  '/components/web/navigation-bar': ROUTES.NAVIGATION_BAR,
-  '/components/web/navigation-stack': ROUTES.NAVIGATION_STACK,
-  '/components/web/nudgers': ROUTES.NUDGER,
-  '/components/web/pagination': ROUTES.PAGINATION,
-  '/components/web/panels': ROUTES.PANEL,
-  '/components/web/popovers': ROUTES.POPOVER,
-  '/components/web/progress': ROUTES.PROGRESS,
-  '/components/web/section-list': ROUTES.SECTION_LIST,
-  '/components/web/sliders': ROUTES.SLIDER,
-  '/components/web/spinners': ROUTES.SPINNER,
-  '/components/web/star-rating': ROUTES.STAR_RATING,
-  '/components/web/tables': ROUTES.TABLE,
-  '/components/web/text': ROUTES.TEXT,
-  '/components/web/tickets': ROUTES.TICKET,
-  '/components/web/tooltips': ROUTES.TOOLTIP,
-  '/components/web/typography': ROUTES.TEXT,
-  '/tokens/layout': ROUTES.SPACINGS,
-  '/tokens/colors': ROUTES.COLORS,
-  '/using/contributing': ROUTES.USING_BACKPACK,
-  '/style-guide/copywriting': ROUTES.GENERATED_GUIDELINES_COPYWRITING,
   '/style-guide/tone-of-voice': ROUTES.TONE_OF_VOICE,
```

Comment:
I left one in to aid self documentation, so when we need to add new ones in future it's clear how they work.